Makefile

```diff
@@ -193,6 +229,23 @@ KUSTOMIZE = $(shell which kustomize)
 endif
 endif
 
+.PHONY: protoc-gen-go
+protoc-gen-go: $(PROTOC_GEN_GO) ## Download protoc-gen-go locally if necessary.
+$(PROTOC_GEN_GO): $(LOCALBIN)
+	GOBIN=$(LOCALBIN) go install google.golang.org/protobuf/cmd/protoc-gen-go@$(PROTOC_GEN_GO_TOOLS_VERSION)
+
+.PHONY: protoc
+protoc: $(PROTOC) ## Download protoc locally if necessary.
+$(PROTOC): $(LOCALBIN)
+	wget https://github.com/protocolbuffers/protobuf/releases/download/v$(PROTOC_VERSION)/protoc-$(PROTOC_VERSION)-linux-x86_64.zip
+	unzip -DD protoc-$(PROTOC_VERSION)-linux-x86_64.zip bin/protoc && rm protoc-$(PROTOC_VERSION)-linux-x86_64.zip
```

Comment:
If we use the same approach as the `oc` and `operator-sdk` targets the binary won't be downloaded every time.

Reply:
oh yes, I didn't see we already have something in place for bin.
Do we need to check for system `protoc`  like this?
https://github.com/rigazilla/gingersnap-operator/blob/0cc57fd6f9e1c2ecd9fc9f9ca740c7d4d594fb74/Makefile#L312-L313

Having protoc installed is quite common, but maybe with different version

Reply:
How problematic do you think different versions would be?

It's a small download, so I don't mind either way

Reply:
let's download our version